### PR TITLE
Chutney obfs4

### PR DESCRIPTION
**Pull request by nmathewson**

### CODE REVIEW

File: lib/chutney/TorNet.py [hunk 1]
```diff
@@ -633,11 +640,26 @@ def _getBridgeLines(self):
...
-        bridgelines = "Bridge %s:%s\n" % (self._env['ip'],
-                                          self._env['orport'])
+        if self._env['pt_bridge']:
+            port = self._env['ptport']
+            transport = self._env['pt_transport']
+            extra = self._env['pt_extra']
+        else:
+            port = self._env['orport']
+            transport = ""
+            extra = ''
```
Comment (teor2345) [inline lib/chutney/TorNet.py]: I had to stare at these two lines for a while, before I realised they were both empty strings.

Nitpick: let's use single quotes for both.

Comment (nmathewson) [reply lib/chutney/TorNet.py]: Fixup as f79dedc397609d

Comment (nmathewson) [reply lib/chutney/TorNet.py]: (whoops, didn't notice that you had suggested single quotes. I went with double; I hope that's ok)

**Comment by teor2345** [comment]
Squashed and merged.

**Pull request closed without merging**